.gitignore

```diff
@@ -21,4 +21,6 @@ public
 .hvm
 
 static/output.css
-ensign/.Rhistory
\ No newline at end of file
+ensign/.Rhistory
+
+.tool-versions
```

Comment:
So there's kind of a long story involved in this one but it has to do with the fact that Hugo versions keep causing me build error issues (probably because i build so infrequently now and am probably missing big swaths of updates) and so i've had to do some superglue-ing via bash to get my brew version synced to the one in our GH actions, but i think I've it sorted for now at least 😎